### ISSUE TITLE
Properties/values review for next major version

### ISSUE THREAD

**Issue by ratoaq2** [guessit-io/guessit#459]
A new major version brings an opportunity to review the current properties and their values.
It doesn't mean that everything should change, but most likely somethings could be enhanced.

The first thing that I noticed is that guessit has a lot of properties and a lot of values. Wow.
I decided to visit all properties and their values and build a list with my findings. That could help us to understand and review them.

### episode_details
- [x] `Oav` = `Ova` => should move to `other`: `Original Animated Video`
> Original video animation (オリジナル・ビデオ・アニメーション? Orijinaru bideo animēshon), abbreviated as OVA (オーブイエー / オーヴィーエー / オヴァ? ōbuiē, ōvīē or ova) media (and sometimes as OAV, original animated video), are animated films and series made specially for release in home video formats without prior showings on television or in theatres

- [x] `Omake` => `Extras`
> Omake (御負け?, usually written おまけ) means extra in Japanese

### screen_size
- [x] Detect `360i`, `368i`, `480i`, `576i`, `900i` instead of progressive-scan
- [x] `4K` => `2160p`
> The real solution would have been to abandon the 4K moniker entirely and instead use the designation 2160p. Display and broadcast resolutions have always referred to resolution in terms of horizontal lines, with the letters “i” and “p” referring to interlacing, which skips every other line, and progressive scan, which doesn’t: 576i (PAL), 480i (NTSC), 576p (DVD), 720p, 1080i, 1080p, and so on.

### video_codec
- [x] `h264` => `H.264`
> H.264 or MPEG-4 Part 10, Advanced Video Coding (MPEG-4 AVC)

- [x] `h265` => `H.265`
> High Efficiency Video Coding (HEVC), also known as H.265 and MPEG-H Part 2

- [x] `Mpeg2` => `MPEG-2`
> MPEG-2 (a.k.a. H.222/H.262 as defined by the ITU)

- [x] `Real` => `RealVideo`  (official name is camel case)
> RealVideo is a suite of proprietary video compression formats

- [x] `XviD` => `Xvid`
> Xvid (formerly "XviD")

### video_profile
- [ ] `8bit` => `8-bit`
- [ ] `10bit` => `10-bit`
- [x] `BP` => `Baseline`
- [x] `MP` => `Main`
- [x] `HP` => `High`
- [x] `XP` => `Extended`
- [x] `Hi422P` => `High 4:2:2`
- [x] `Hi444PP` => `High 4:4:4 Predictive`

~~### video_api:~~
~~- [ ] `DXVA` => `DirectX Video Acceleration`~~

### audio_codec:
- [x] `DTS` + `audio_profile`: `HD` or `HDMA` => `DTS-HD`
- [x] `DTS` + `audio_profile`: `HDMA` => `DTS-HD` + `audio_profile: MA`
> See comments in this PR

- [x] `TrueHD` => `TrueHD`
> Dropped Dolby from the name as it's a well known an usual way to refer to Dolby TrueHD
- [ ] `AC3` => `AC-3`
- [ ] `EAC3`=> `Enhanced AC-3`
- [x] `Flac` => `FLAC`
- [x] `DolbyAtmos` => `Atmos`
> Dropped Dolby from the name as it's a well known an usual way to refer to Dolby Atmos

### audio_profile:
- [x] `HD` and `HDMA` => as explained in audio_codec section
- [x] `HE` => `High Efficiency`
- [x] `LC` => `Low Complexity`
- [x] `HQ` => `High Quality`  (It seems that AC3 HQ means encoded in AC-3 using HQ mic)
> it stills the hq mic that was used from previous releases

### size:
- [ ] `1MB` => `<Size: 1024000>`
> The idea is to return an object `Size` (which stores the value in bytes and its string format returns a human-readable format. e.g: `344 MB`). To help when using guessit as an API.

### edition:
- [x] `Collector Edition` => `Collector`
- [x] `Criterion Edition` => `Criterion`
- [x] `Deluxe Edition` => `Deluxe`
- [x] `Special Edition` => `Special`
- [x] `Limited Edition` => `Limited`
- [x] `Theatrical Edition` => `Theatrical`

### other:
- [x] `AudioFix` => `Audio Fixed`
- [x] `CC` => Should move to `edition`: `Criterion`
> Stands for Criterion Collection

- [x] `DDC` => should move to `edition`: `Director's Definitive Cut` 
> Stands for Director's Definitive Cut

- [x] `DualAudio` => `Dual Audio` 
- [x] `Fansub` => `Fan Subtitled`
> A fansub (short for fan-subtitled)

- [x] `Fastsub` => `Fast Subtitled`
- [x] `FINAL` => should move to `episode_details`: `Final`
> Season/Series Final episode

- [x] `FullHD` => `Full HD`
- [x] `HQ` => `High Quality`
- [x] `HR` => `High Resolution`
> HRHD (high-resolution high-definition) (also HR, HRHDTV, or HR.HDTV) is an initialism referring to an image resolution derived from high-definition video

- [x] `LD` => `Line Dubbed`
> LD stands for Line Dubbed.

- [x] `MD` => `Mic Dubbed`
> MD stands for Mic Dubbed.

- [x] `mHD` and `HDLight` => `MicroHD`
> El formato MicroHD (mHD) es un archivo de vídeo creado a partir de otro vídeo en alta definición usando H.264/MPEG-4 AVC como códec de vídeo y AAC como formato de audio

- [x] `LiNE` => `Line audio`
- [x] `OV` + subs => `Hardcoded Subtitles`
> Over Video subtitles

- [x] `VO` or `OV` => `Original Video`
> Version originale

- [x] `R5` => `Region 5`
> The R5 is a retail DVD from region 5

- [x] `ReEncoded` => `Reencoded`
- [x] `SyncFix` => `Sync Fixed`
- [x] `UltraHD` => `Ultra HD`
- [x] `WideScreen` => `Widescreen`

Again, my proposal is not to change everything, but to come up with a list that can give us information and visibility, so it can be reviewed.

Changes after review on 02.06.2017 19:30 CEST
- `OVA` move to other
- Keep `DXVA` acronym (it's a well known acronym)
- `H.264` / `H.265` / `RealVideo`
- `DTS-HD MA` should be an `audio_codec` since `HD` and `HDMA` are not `audio_profile`s
- `size` property to return a proper object
- Detect `Director's Definitive Cut` instead of just `Director's Cut`

Changes on 04.06.2017 07:50 CEST
- Proposal for DTS, DTS-HD and DTS-HD MA
- Audio codec changes. Tried to have shorten versions for them

**Comment by Toilal** [comment]
I agree with all of those changes, great job !

**Comment by ratoaq2** [comment]
I'm still reviewing this initial list. 
For instance:
`Original Animated Video` probably should move to `other` instead of episode details 

I didn't like `DirectX Video Acceleration`. Too verbose. Ofc, just my opinion, so it's subjective. 

The video codecs, `h264` is actually `H.264`. Same for `h265`. 

`Dolby Atmos` and has no other name or single word label. `E-AC-3` became enigmatic. But if we change, probably `AC-3` should change as well. 

Anyway. I'll be continuing working on this

**Comment by ratoaq2** [comment]
@Toilal I did a 2nd pass review. I marked the changes in the first post. I checked the changes that feels right to me. And I left unchecked the ones that I'm still unsure. Most of it is around `audio_codec` and whether to use acronyms or official names. I don't know why but I would prefer to have all of them with well-known acronyms, but the new ones `Dolby Atmos`, `Dolby TrueHD` and `DTS-HD Master Audio` don't have a simple and less verbose version. :-/

**Comment by ratoaq2** [comment]
`DTS-HD` and `DTS-HD MA` are two different things: `DTS-HD High Resolution Audio` and `DTS-HD Master Audio`. All of them has a compatible `DTS` soundtrack for backwards compatibility: https://en.wikipedia.org/wiki/DTS_(sound_system)#DTS-HD_High_Resolution_Audio

So it's a `DTS` extension and current behaviour is perfect: Detecting it as a audio_profile. Here's a media info for a `DTS-HD MA`

```
Audio #1
ID                                       : 2
Format                                   : DTS
Format/Info                              : Digital Theater Systems
Format profile                           : MA / Core
Mode                                     : 16
Format settings, Endianness              : Big
Codec ID                                 : A_DTS
Duration                                 : 0 h 2 min
Bit rate mode                            : Variable / Constant
Bit rate                                 : Unknown / 1 509 kb/s
Channel(s)                               : 8 channels / 6 channels
Channel positions                        : Front: L C R, Side: L R, Back: L R, LFE / Front: L C R, Side: L R, LFE
Sampling rate                            : 48.0 kHz
Frame rate                               : 93.750 FPS (512 spf)
Bit depth                                : 24 bits
Compression mode                         : Lossless
```

**Comment by ratoaq2** [comment]
Sorry the spam, but what I said wasn't actually accurate.
`DTS-HD High Resolution Audio` (`DTS-HD`) and `DTS-HD Master Audio` (`DTS-HD MA`) are indeed backwards compatible with `DTS`, but they are one new codec: `DTS-HD` with has 2 profiles: `Core` and `MA`:

Complete media info with only the relevant parts:
```
Audio #1
Kind of stream                           : Audio
Stream identifier                        : 0
Stream identifier                        : 1
StreamOrder                              : 1
Inform                                   : English, Unknown / 1 509 kb/s, 48.0 kHz, 24 bits, 8 channels / 6 channels, DTS (MA / Core)
ID                                       : 2
Format                                   : DTS
Format/Info                              : Digital Theater Systems
Commercial name                          : DTS
Format profile                           : MA / Core
Mode                                     : 16
Format settings, Endianness              : Big
Codec ID                                 : A_DTS
Codec                                    : DTS-HD
Codec/Family                             : DTS
Bit rate mode                            : Variable / Constant
Bit rate                                 : Unknown / 1 509 kb/s
Channel(s)                               : 8 / 6
Channel(s)                               : 8 channels / 6 channels
Compression mode                         : Lossless / Lossy
Compression mode                         : Lossless / Lossy
```

As you can see the codec is `DTS-HD` and the two profiles: `Core` and `MA` for each one. It uses the same codec id as `DTS` to main the backwards compatibility since it's main soundstream is a regular `DTS` stream. The second soundstream has the additional sound data (extension).

> DTS-HD is lossy, supports up to 5.1 channels, and has a sampling rate of up to 96kHz. DTS-HD MA, on the other hand, is lossless, supports up to 7.1 channels, and has a sampling rate of up to 192kHz
Whitepapers: https://resources.dts.com/bp/#/folder/1494135

Wikipedia wasn't so accurate, so I was digging a bit more in some expert forums and sites.

**Comment by ratoaq2** [comment]
Fixed by #464
